src/main/java/core/basesyntax/service/ParsingReportService.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.service;
+
+import core.basesyntax.db.FruitShopStorage;
+
+public interface ParsingReportService {
+    String getValueToWrite(FruitShopStorage fruitShopStorage);
```

Comment:
```suggestion
    String createReport();
```
pass storage in constructor when creating this service